Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -202,16 +202,37 @@ Location-based measures the grid carbon intensity (annual average) of a regional
 The only figure that matters if you’re trying to optimize the scheduling of your compute in real-time is the marginal emissions intensity. This is the emissions intensity of the marginal power plant which will be turned up if you schedule some compute (e.g. increase electricity demand from the grid) at that moment.
 
 ### Embodied Emissions  (`M`) 
-Embodied carbon (otherwise referred to as "Embedded Carbon") is the amount of carbon pollution emitted during the creation and disposal of a hardware device. When calculating the carbon emissions for software, include the embodied carbon of the supporting infrastructure inside your software boundary.
+Embodied carbon (otherwise referred to as “Embedded Carbon”) is the amount of carbon emitted during the creation and disposal of a hardware device. 
 
-`M = TE * TR/EL * RU`
+When software runs on a device, a fraction of the total embodied emissions of the device is allocated to the software. This is the value of `M` that you need to calculate in the SCI equation. 
+
+That fraction consists of both a time-share and a resource-share. The length of time your software runs on the device determines the time-share. The percentage of the device reserved just for your application during the time-share determines your resource-share.
```

Comment:
```suggestion
That fraction consists of both a time-share and a resource-share. The length of time your software runs on the device determines the time-share. The percentage of the device reserved just for your application during the time-share correlates with your resource-share.
```

Reply:
Reason for amending the wording is that there is a correlation between the two and not a strict determination.

Reply:
I'm trying to define the term resource-share which is "The percentage of the device reserved just for your application during the time-share", so correlates doesn't quite match up to the intent of that sentence.

Perhaps it should be more direct like so:

"The percentage of the device reserved just for your application during the time-share **is** your resource-share."